File: Ywy/JavaScript/level1/test/EvenAndOdd.test.js
import EvenAndOdd from '../EvenAndOdd';

test('2 is "Even"', () => {
    expect(new EvenAndOdd().solution(2)).toEqual("Even");
});

test('1 is Odd', () => {
    expect(new EvenAndOdd().solution(1)).toEqual("Odd");
});

File: Ywy/JAVA/src/level1/test/cases/EvenAndOddTest.java
package level1.test.cases;

import static org.junit.Assert.assertEquals;

import org.junit.Before;
import org.junit.Test;

import level1.EvenAndOdd;

public class EvenAndOddTest {
	private int[] nums;
	private String[] answers;
	private EvenAndOdd evenAndOdd;
	
	@Before
	public void setUp() {
		nums = new int[] {2, 1};
		answers = new String[] {"Even", "Odd"};
		evenAndOdd = new EvenAndOdd();
	}
	
	@Test
	public void test() {
		int index = 0;
		int range = nums.length;
		do {
			assertEquals(evenAndOdd.solution(nums[index]), answers[index]);
		} while(++index < range);
	}

}
